### ISSUE TITLE
Custom `STON-Core` package is not kosher

### ISSUE THREAD

**Issue by dalehenrich** [hpi-swa/smalltalkCI#108]
The custom `STON-Core` package for Squeak and Pharo means that it is impossible to use an alternate or different version of STON.

When SmalltalkCI is loaded into a Pharo image, the SmalltalkCI `STON-Core` package unconditionally replaces whatever may have been in that package previously ... 

While the SmalltalkCI `STON-Core` may be functional in Pharo, it differs quite a bit from the officially released version of `STON-Core` (`STON-Core-SvenVanCaekenberghe.70` in http://smalltalkhub.com/mc/SvenVanCaekenberghe/STON/main).

The proper way to reference STON (in Pharo at least) is to reference the ConfigurationOfSton as follows:

```Smalltalk
  spec
    for: #'pharo'
    do: [ 
      spec
        configuration: 'Ston'
        with: [ 
          spec
            version: #'stable';
            loads: 'Core';
            repository:
                'http://smalltalkhub.com/mc/SvenVanCaekenberghe/STON/main' ].
      spec
        package: 'SmalltalkCI-Core'
          with: [ 
              spec
                requires: #('Ston');
                includes: #('SmalltalkCI-Pharo' 'Ston') ];
        package: 'SmalltalkCI-Pharo'
          with: [ spec requires: #('SmalltalkCI-Core') ] ].
```

Technically the same types of problems would exist for Squeak as I don't think there is an official port of STON to Squeak(?), but once an official port is done for Squeak, this package will cause similar problems ....

I hit this while working on Issue #105 and will patch it there, but I can submit a separate PR, if you want to isolate the changes for this issue separately ...

**Comment by dalehenrich** [comment]
The following commits are relevant to this issue:  	2ec4808,  77d1185, and  dc65962

**Comment by fniephaus** [comment]
Thanks, @dalehenrich! Happy to merge this into `master` once it's ready
